conda-forge.yml

```diff
@@ -10,9 +10,12 @@ skip_render:
 provider:
   linux_64: circle
   osx_64: azure
+  osx_arm64: azure
   win: None
   linux_ppc64le: None
   linux_aarch64: None
+build_platform:
+  osx_arm64: osx_arm64
```

Comment:
conda-forge has `build_platform.osx_arm64 = osx_64` by default and if you don't include this, you don't get the additional config when you rerender.

Still need to see if the rendered files will correctly build on an `osx_arm64` agent.  I suspect that they may fail spectacularly because native osx_arm64 doesn't seem to be fully supported by Azure Pipelines yet. ( see https://github.com/microsoft/azure-pipelines-agent/issues/3184)

Reply:
I did some research yesterday and found that Github Actions supports arm64 on "Larger Instances" that Teams or Enterprise accounts can use but not on the free ones for open-source.  Azure Pipelines only supports them for self-hosted runners.